embedded-aerospike/src/main/java/com/playtika/testcontainer/aerospike/AerospikeProperties.java

```diff
@@ -11,7 +11,7 @@
 @ConfigurationProperties("embedded.aerospike")
 public class AerospikeProperties extends CommonContainerProperties {
 
-    static final String BEAN_NAME_AEROSPIKE = "aerospike";
+    public static final String BEAN_NAME_AEROSPIKE = "aerospike";
```

Comment:
why do you need public? I dont see usages

Reply:
usage on other library